app/questionnaire/questionnaire_store_updater.py

```diff
@@ -16,15 +19,15 @@
 class QuestionnaireStoreUpdater:
     """Component responsible for any actions that need to happen as a result of updating the questionnaire_store"""
 
-    EMPTY_ANSWER_VALUES: Tuple = (None, [], "", {})
+    EMPTY_ANSWER_VALUES: tuple[None, list, str, dict] = (None, [], "", {})
```

Comment:
@petechd - I think the comments are against a prior commit, should all be ok here?

Reply:
@katie-gardner thanks somehow I didn't see them, all good.